pyproject.toml

```diff
@@ -3,10 +3,19 @@ name = "ocpp"
 version = "0.2.0"
 description = "Python package implementing the JSON version of the Open Charge Point Protocol (OCPP)."
 authors = [
-	"Auke Willem Oosterhoff <aukewillem.oosterhoff@mobilityhouse.com>"
+	"Andre Duarte <andre.duarte@mobilityhouse.com>",
+	"Auke Willem Oosterhoff <aukewillem.oosterhoff@mobilityhouse.com>",
+	"Greg Lutostanski <greg.lutostanski@mobilityhouse.com>",
+	"Jonathan Herrmann <jonathan.herrmann@mobilityhouse.com>",
+	"Laysa Uchoa <laysa.uchoadasilva@mobilityhouse.com>",
+	"Oz N Tiram <oz.tiram@mobilityhouse.com>",
```

Comment:
Oz does not work for Mobility house anymore...does it make sense to leave it here?

Reply:
I think it is fair to list everyone who worked on it. We could remove the email address though. That is likely not going to be an active address.

Reply:
yup, agree!

Reply:
Maybe its better to link github accounts or his github at least?

Reply:
yup, mabe that it the most stable option...what do you think @OrangeTux ?